source/dev_guide/testing.rst

```diff
@@ -0,0 +1,73 @@
+Testing E3SM Diagnostics
+==============================================
+
+Unit tests
+----------
+
+Run all unit tests by doing the following:
+
+    .. code::
+
+        pip install . # Install your changes
+        ./tests/test.sh # Run all unit tests
+
+Automated tests
+---------------
+
+We have a Continuous Integration / Continuous Delivery (CI/CD) workflow.
+The unit tests are run automatically as part of this.
```

Comment:
@tomvothecoder I figure you can update this part once #385 merges.

Reply:
Makes sense, will do after merging that PR